templates/template_vars.yaml

```diff
@@ -1,7 +1,7 @@
 General:
-  release: 24.08
-  triton_cli_version: 0.0.11
-  genai_perf_version: 0.0.6dev
+  release: 24.09
+  triton_cli_version: 0.0.12
```

Comment:
just to confirm, will triton cli version be bumped to 0.0.12 along with 24.09 release? (I don't see any PR or branch for the change in the triton cli repo)

Reply:
triton_cli_version will be updated later.

Reply:
should we wait for the 24.09 image so we can update both triton cli version and trtllm version at the same time?

Reply:
Also this PR will be merged after 24.09 containers are published.